src/lib/OpenEXRCore/internal_ht_common.cpp

```diff
@@ -147,14 +143,14 @@ make_channel_map (
 
         int avail_cs_i = 3;
         int offset = 0;
-        for (size_t file_i = 0; file_i < channel_count; file_i++)
+        for (size_t file_i = 0; file_i < static_cast<size_t>(channel_count); file_i++)
```

Comment:
channel_count is int. If you change this to `int file_i = ...`, you don't need the casts at all.

Reply:
done